code/modules/hydroponics/grown.dm

```diff
@@ -42,6 +42,8 @@
 		seed.prepare_result(src)
 		transform *= TRANSFORM_USING_VARIABLE(seed.potency, 100) + 0.5 //Makes the resulting produce's sprite larger or smaller based on potency!
 		add_juice()
+		if(seed.variant && seed.variant != "")
```

Comment:
A string of `""` evalutes as false.
![image](https://user-images.githubusercontent.com/25063394/129193224-eab17828-7f65-4990-bb5f-ecd8af6e63ae.png)
The second check here (and all other places) is redundant.

Reply:
Done